swift/ql/test/library-tests/dataflow/dataflow/DataFlowInline.expected

```diff
@@ -0,0 +1 @@
+| test.swift:497:17:497:17 | .str | Unexpected result: flow=496 |
```

Comment:
It looks like this test is already passing _before_ any dataflow library changes? Are we sure we don't actually have all the required flow already?